infrastructure/template.yaml

```diff
@@ -1788,6 +1813,29 @@ Resources:
 
   ###############################################################################
 
+  BadEventBridgeStateMachineExecutionRole:
+    Type: AWS::IAM::Role
```

Comment:
Can you add a dev-like condition to this please?

Reply:
I have removed `BadEventBridgeStateMachineExecutionRole` it done it's job